tests/classifiers/test_bisac.py

```diff
@@ -25,100 +22,100 @@ class TestMatchingRule(object):
+        assert None == rule.match("good", "food")
+        assert None == rule.match("fads")
 
     def test_special_tokens_must_be_first(self):
         # In general, special tokens can only appear in the first
         # slot of a ruleset.
         for special in (juvenile, fiction, nonfiction):
-            assert_raises(ValueError, MatchingRule, True, "first item", special)
+            pytest.raises(ValueError, MatchingRule, True, "first item", special)
 
         # This rule doesn't apply to the 'anything' token.
         MatchingRule(True, "first item", anything)
 
     def test_juvenile_match(self):
         rule = MatchingRule(True, juvenile, "western")
-        eq_(True, rule.match("juvenile fiction", "western"))
-        eq_(None, rule.match("juvenile nonfiction", "western civilization"))
-        eq_(None, rule.match("juvenile nonfiction", "penguins"))
-        eq_(None, rule.match("young adult nonfiction", "western"))
-        eq_(None, rule.match("fiction", "western"))
+        assert True == rule.match("juvenile fiction", "western")
+        assert None == rule.match("juvenile nonfiction", "western civilization")
+        assert None == rule.match("juvenile nonfiction", "penguins")
+        assert None == rule.match("young adult nonfiction", "western")
+        assert None == rule.match("fiction", "western")
 
     def test_ya_match(self):
         rule = MatchingRule(True, ya, "western")
-        eq_(True, rule.match("young adult fiction", "western"))
-        eq_(True, rule.match("young adult nonfiction", "western"))
-        eq_(None, rule.match("juvenile fiction", "western"))
-        eq_(None, rule.match("fiction", "western"))
+        assert True == rule.match("young adult fiction", "western")
+        assert True == rule.match("young adult nonfiction", "western")
+        assert None == rule.match("juvenile fiction", "western")
+        assert None == rule.match("fiction", "western")
 
     def test_nonfiction_match(self):
         rule = MatchingRule(True, nonfiction, "art")
-        eq_(True, rule.match("juvenile nonfiction", "art"))
-        eq_(True, rule.match("art"))
-        eq_(None, rule.match("juvenile fiction", "art"))
-        eq_(None, rule.match("fiction", "art"))
+        assert True == rule.match("juvenile nonfiction", "art")
+        assert True == rule.match("art")
+        assert None == rule.match("juvenile fiction", "art")
+        assert None == rule.match("fiction", "art")
 
     def test_fiction_match(self):
         rule = MatchingRule(True, fiction, "art")
-        eq_(None, rule.match("juvenile nonfiction", "art"))
-        eq_(None, rule.match("art"))
-        eq_(True, rule.match("juvenile fiction", "art"))
-        eq_(True, rule.match("fiction", "art"))
+        assert None == rule.match("juvenile nonfiction", "art")
+        assert None == rule.match("art")
+        assert True == rule.match("juvenile fiction", "art")
+        assert True == rule.match("fiction", "art")
```

Comment:
Looking at the Pytest website it does seem clear that the syntactic sugar has gone away, but how bad is it? When the tests are running and one of these assertions fail, do we just get a bare AssertionError or does it have info about what went wrong?

Reply:
I find the output from pytest very nice. Here is an example of an [old failed test run](https://github.com/lyrasis/simplye-server-core/runs/1990635874?check_suite_focus=true#step:8:1507). 

There are some technical details in the [pytest documentation](https://docs.pytest.org/en/stable/assert.html#assertion-introspection-details) about how they achieve the introspection into what went wrong.